Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```diff
@@ -529,6 +530,14 @@ def upload_indicators_list_request(self, reference_name, indicators_list):
             "POST", url, params=params, data=json.dumps(indicators_list)
         )
 
+    def get_custom_fields(
+            self, limit: Optional[int] = None) -> List[dict]:
+        url = urljoin(self._server, "api/config/event_sources/custom_properties/regex_properties")
+        headers = self._auth_headers
+        if limit:
```

Comment:
i dont see you pass `limit` to the function, so either do so or remove it

Reply:
it's ok. It'll be used in the next enhancement ill open pr for today.